ntoskrnl/ke/amd64/cpu.c

```diff
@@ -52,14 +52,15 @@ KiSetProcessorType(VOID)
      * Model and the Step, while the Type contains the returned Type.
      * We ignore the family.
      *
-     * For the stepping, we convert this: zzzzzzxy into this: x0y
+     * For the stepping, we convert this: zzzxzzxy into this: xx0y
      */
     Stepping = CpuInfo.Eax & 0xF0;
     Stepping <<= 4;
-    Stepping += (CpuInfo.Eax & 0xFF);
-    Stepping &= 0xF0F;
+    Stepping += ((CpuInfo.Eax & 0x000F0000) >> 4);
+    Stepping += (CpuInfo.Eax & 0xF);
```

Comment:
Nit pick:
```suggestion
    Stepping |= (CpuInfo.Eax & 0x0F);
```
While here, everywhere, `|=` seems more consistent.
Are the outer parentheses really needed?

Reply:
Guidance on nit picks: if they're the only thing you have to contribute to a PR, stay quiet.